README.md

```diff
@@ -42,10 +42,11 @@ $CFG->curlsecurityallowedport
 Branches
 --------
 
-| Moodle verion     | Branch      | PHP       | Backports |
-| ----------------- | ----------- | --------  | ----------|
-| Moodle 2.7 to 3.9 | master      | 5.5 - 7.4 | MDL-70649 |
```

Comment:
why did yo remove the backport column?

Also does this plugin work under php 5.5? Given that all of the branches map to master I think we can stick with just a single row

Reply:
Have changed back to single row - adding Totota version details.

Reply:
Can you please make this look good as either raw markdown or as processed markdown into html, ie add some whitespace to make the table readable

Reply:
Also because we have a moodle column and a totara column you can probably drop that from the table cells

Reply:
Sorry when you are saying "Also because we have a moodle column and a totara column you can probably drop that from the table cells", do you mean by dropping Backports column?

Reply:
No I mean drop the word 'moodle' from inside the data just to make it less wide

```
| Moodle 2.7 to 3.9 |
vs just
| 2.7 to 3.9 |
```

Reply:
Right. Please check again.

Reply:
Also I'm quite suspicious of the versions in the table, have you actually tested this in 2.7 or on php 5? I'm not fussed either way, we don't need to support back that far. But whatever appear in this table should actually have been tested and known to work.

Reply:
Hi Bredan please check again.